tests/http_plugin_test.py

```diff
@@ -35,21 +37,34 @@
     TestHelper.printSystemInfo("BEGIN")
 
     Print("Stand up cluster")
-    # standup cluster with HTTPS enabled, but not configured
-    # HTTP should still work
-    extraArgs={ 0 : "--https-server-address 127.0.0.1:5555" }
-    # specificExtraNodeosArgs=extraArgs
 
-    if cluster.launch(dontBootstrap=True, loadSystemContract=False) is False:
+    if cluster.launch(dontBootstrap=True, loadSystemContract=False, specificExtraNodeosArgs = {0: "--http-validate-host true"}) is False:
```

Comment:
Do you really need `--http-validate-host true` here? It's the default.

Reply:
Yes, I have to. Somewhere in the TestHarness changed it to false.

Reply:
ah okay, it's interesting because you can't `--http-validate-host true --http-validate-host true` nor
```
http-validate-host = true 
http-validate-host = true 
```

But you can `http-validate-host = true` & `--http-validate-host true`